cfme/tests/infrastructure/test_vm_retirement_rest.py

```diff
@@ -18,6 +20,11 @@
 ]
 
 
+@pytest.fixture
+def retire_action(appliance):
```

Comment:
I don't think this should be a fixture, as its not at all related to setup.  It should just be picked within the test case, even though that will repeat a single line of code in both test cases.